chia/cmds/wallet_funcs.py

```diff
@@ -216,7 +216,8 @@ async def send(args: dict, wallet_client: WalletRpcClient, fingerprint: int) ->
 
 async def get_address(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
     wallet_id = args["id"]
-    res = await wallet_client.get_next_address(wallet_id, False)
+    new_address: bool = args.get("new_address", False)
```

Comment:
Is the `"new_address"` key going to be any less likely to be present than `"id"`?

Though, the whole `execute_with_wallet()` could just be a context manager probably which would allow elimination of this indirection of parameters through a dict entirely.  But that's entirely unrelated to this PR...

Reply:
I can imagine `get_address` being called elsewhere. I agree that `execute_with_wallet` could be updated outside of this PR.

Reply:
I hope we don't leak the `args: dict` interface out for other use.  But yeah, that can be dealt with independently if any of us choose to.